services/motion/builtin/move_request.go

```diff
@@ -33,6 +33,7 @@ const (
 	defaultReplanCostFactor = 1.0
 	defaultMaxReplans       = -1 // Values below zero will replan infinitely
 	baseStopTimeout         = time.Second * 5
+	poleEpsilon             = 0.0001 // If the base is this close to vertical, we cannot plan.
```

Comment:
Why is this a separate variable from https://github.com/viamrobotics/rdk/pull/3531/files#diff-545046245b3c4d0b8712109df981910ea6e62533cd7bcaa4cd8e75c83090d835R17?

Should we just use: https://github.com/viamrobotics/rdk/pull/3531/files#diff-545046245b3c4d0b8712109df981910ea6e62533cd7bcaa4cd8e75c83090d835R17?

Reply:
These are measuring two conceptually different things. The location at which we swap how we compute theta isn't really related to whether the motion service considers a base to be "pointing up or down". Yes, they are both the same value, and yes, they both measure the distance of an orientation vector's OZ from 1, but I think to use the same constant here would be misleading to a reader. They would look up `OrientationVectorPoleRadius`, see the documentation, and think that that applied to its use here, when it does not.